sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -43,6 +43,7 @@ public void FormRecognizerClientCannotAuthenticateWithFakeApiKey()
         }
 
         [Test]
+        [Ignore("ADD not supported")]
```

Comment:
Should it be AAD?

By the way, are we not supporting AAD anymore for FR?

Reply:
lol. yes. thanks!
2.1-preview.2 doesn't support AAD yet so decided to ignore the test until the service can fix it